src/snowcli/cli/common/flags.py

```diff
@@ -1 +1,48 @@
+from __future__ import annotations
+
+import typer
+from snowcli.utils import check_for_connection
+
 DEFAULT_CONTEXT_SETTINGS = {"help_option_names": ["-h", "--help"]}
+
+
+ConnectionOption = typer.Option(
+    "dev",
+    "-c",
+    "--connection",
+    "--environment",
+    help="Connection / environment name",
+    callback=check_for_connection,
+    is_eager=True,
+)
+
+AccountOption = typer.Option(
+    None,
+    "-a",
+    "--accountname",
+    "--account",
+    help="Name assigned to your Snowflake account.",
+)
+
+UserOption = typer.Option(
+    None, "-u", "--username", "--user", help="Username to connect to Snowflake."
+)
+
+PasswordOption = typer.Option(
+    ...,
```

Comment:
Why `...`?

Reply:
It's refactor, I haven't used it. But I can change it to None

Reply:
It's refactor but the goal of this refactor is to standardise the flags.
IMO lack of explicit flag names allows to have different flags to pass password value what IMO should be standardised.

I'm unresolving conversation.